### DISCUSSION TITLE
Welcome to plyr-react Discussions!

### DISCUSSION THREAD

**Discussion by chintan9** [chintan9/plyr-react#556] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sreekeshiyer** [comment]
Hey!
I'm using YouTube videos as cards on my webpage (NextJs)
![image](https://user-images.githubusercontent.com/67017933/137196083-6b079a5b-ac4b-43c2-8119-a7006372f675.png)

Is there a way I can hide the video controls just on the thumbnail? I would want them back once the user clicks on the video. Is it possible?

**Reply by realamirhe** [reply]
Hi @sreekeshiyer , 
The most easiest and performant way, would be to do it in css, it also has animation

```css
.plyr__controls {
    opacity: 0;
}
.plyr:hover .plyr__controls {
    opacity: 1;
}
```

It seems not to work perfect with props changes and component remounting phase as mentioned in #654.
So till next fix, It would be (humble) hack to do what you want

**Note**: you asked for click event but I posted the hover action, for click event you must add className  to plyr or its wrapper and add corresponding css to your stlye, but make sure to not make component unmounted.
You can use low-level Js document selectors (e.g. querySelector), or working with Ref for plyr instance (`ref.current.plyr`) and memoizing your component not being re-render.

**Reply by sreekeshiyer** [reply]
Thank you :)
